strings/check_isogram.py

```diff
@@ -0,0 +1,28 @@
+"""
+wiki: https://en.wikipedia.org/wiki/Heterogram_(literature)#Isograms
+"""
+
+
+def check_isogram(string: str) -> bool:
+    """
+    An isogram is a word in which no letter is repeated.
+    Examples of isograms are uncopyrightable and ambidextrously.
+    >>> check_isogram('uncopyrightable')
+    True
+    >>> check_isogram('allowance')
+    False
+    """
+    letters = sorted(string)
+
+    for idx, letter in enumerate(letters):
+        if letter == letters[idx - 1]:
+            return False
+
+    return True
+
+
+if __name__ == "__main__":
+    input_str = input("Enter a string ").strip().lower()
```

Comment:
The `str.lower()` should happen ___in___ the function.

Reply:
@cclauss Lowercasing string inside function.